demo/App.vue

```diff
@@ -348,7 +357,7 @@ export default {
     detailsList() {
       if (this.selected.length) {
         return this.selected;
-      } else if (this.location && this.location._id) {
+      } if (this.location && this.location._id) {
```

Comment:
wait huh?

Reply:
Yes, this was auto-linted away due to the `return`.